File: Semester 3/Advanced Programing Methods/JavaExam/src/AppSim/Student.java
package AppSim;

import Domain.IdEntity;

public class Student implements IdEntity<String> {

    private String id, name, mail, teacher;
    private int group;

    public Student(String id, String name, String mail, String teacher, int group) {
        this.id = id;
        this.name = name;
        this.mail = mail;
        this.teacher = teacher;
        this.group = group;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMail() {
        return mail;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    public String getTeacher() {
        return teacher;
    }

    public void setTeacher(String teacher) {
        this.teacher = teacher;
    }

    public int getGroup() {
        return group;
    }

    public void setGroup(int group) {
        this.group = group;
    }
}

File: Semester 3/Advanced Programing Methods/CSApp/CSApp/Domain/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSApp.Domain
{
    public class Student : IHasId<string>
    {
        
        public string id;
        public string Name { get; set; }
        public int Group { get; set; }
        public string Mail { get; set; }
        public string Teacher { get; set; }


        public Student(string id, string name, int group, string mail, string teacher)
        {
            this.id = id;
            this.Name = name;
            this.Group = group;
            this.Mail = mail;
            this.Teacher = teacher;
        }
        
        public string GetId()
        {
            return this.id;
        }

        public void SetId(string Id)
        {
            this.id = id;
        }
    }
}
